collector/pg_process_idle.go

```diff
@@ -52,11 +52,12 @@ func (PGProcessIdleCollector) Update(ctx context.Context, instance *instance, ch
 				SELECT
 				state,
 				application_name,
+    			usename,
```

Comment:
The indentation is not correct here.

Reply:
Sorry for that. But I fixed that in force-push. Am I right?
Thank for your time.